### DISCUSSION TITLE
Help Wanted?

### DISCUSSION THREAD

**Discussion by crazyLillo** [tesivo/tesivo_s_VTMB_italian_translation_for_Unofficial_Patch#42] [Generale]
ciao a tutti
per caso passando da games translator, ho letto l'appello.
io sono disponibile ad aiutarvi
su games translator ho gia scritto quello che saprei/potrei fare
non ci credo che dopo tutto sto tempo ci sia qualcuno che se ne occupa ancora.
io manco da un bel pò di tempo, forse troppo
bhe! io ci sono
l'unica cosa : mi sento un po obsoleto, datemi tempo che mi ambiento
(moltissimo tempo che non uso un pc...)
ciao

**Comment by tesivo** [comment]
Grandioso! Ultimamente sono parecchio preso con la tesi, quindi mi scuso per non aver visto la tua adesione :) Comunque sia ho in mente tre fasi:
-Finire completamente o quasi la 10.7.
-Aggiornare la 10.7 alla 10.9-
-Offrire un setup di installazione della traduzione.

Se vuoi puoi iniziare segnalando ciò che non è ancora tradotto, o che può avere una migliore traduzione, e quindi offrire la tua traduzione. Puoi vedere come viene svolta la traduzione attraverso la lettura dei commits.

Ci sentiamo. A presto. Buon anno!

**Comment by crazyLillo** [comment]
ciao

non si perchè man onriesco ad entrare in gitub, mi chiede un ulteriore codi
che mi inviano per email, lo inserisco, ma dice errato, ne chiedo un altro,
ma anche gli altri danno errore, booo
ok andando al sodo...
mi sta bene tutto quello che vuoi fare, io ti aiuterò nel limite del mio
possibile.
ma ho dei limiti per la mia obsolescenza, (quasi una 15 ina d'anni che non
uso un pc dammi tempo)
al tempo del commodore sproteggevo qualsiasi gioco e un po di dimestichezza
mi è rimasta
in certe cose sono ultaferratissimo e in altre una schiappa
l'età avanza e la barba si fa sempre piu bianca

per esempio, che cosè un commits? nel senso come faccio a leggerli?
pensa che ci ho messo un bel po per registrarmi e capire come postare, ma
c'è l'ho fatta piano piano.

Allora, devi sapere che questo gioco è stata un pò la mia maledizione (ma a
quanto vedo in giro, non solo mia...).
la traduzione di questo gioco è sempre stata problematica per via della
slang usato a Los Angeles.
(io ci ho vissuto 3 anni...)

sto leggendo a piu non posso in giro
non sono riuscito ancora a capire, di quante patch esistano e quante mod
e quali siano le migliori (magari da tradurre in futuro)

ho visto le traduzioni fatte, io ne conosco almeno 3 (multi, Tir, Lunatics).
francamente sono molto perplesso di moltissime stringhe tradotte alla bella
e buona con aggiunte inesistenti e
e ommissioni di pezzi di frase in inglese.
queste per me sono solo interpretazioni di una traduzione
non c'è atmosfera, non c'è pathos nelle traduzioni italiane, quella che
invece c'è nel gioco in lingua originale.

io se lo vuoi sapere, visto che adesso faccio parte del tuo team, ho un
progetto in testa da tanto tempo.
ok tradurre questo gioco a fondo una volta per tutte
poi come ho detto ho un progetto tutto mio di un possibile doppiaggio
soltanto dell'intero tutorial con jack.
molto tempo fa ci provai a farne un pezzo in dialetto romanesco e un pezzo
in napoletano stretto, era a mò di parodia, ma divertente.

ma pensiamo prima alla traduzione
io non so cosa è stato tradotto o no, perchè non lo sto giocando, ho fatto
solo un pezzettino all'inizio per vedere se funzionava tutto.
lo giocherò solo alla fine, quando sarà completo.

fammi solo capire come faccio a vedere i file e i lavori di traduzione e
come faccio a metterci le mani

...è arrivato il momento giusto per Bloodlines

Buon Anno!

**Comment by NarakuITA** [comment]
Vai nella sezione "CODE" li troverai tutte le cartelle e i relativi file, in "lavorazione" o "completati". Ci clicchi e ti si apre il file e puoi vedere cosa è stato tradotto e no tradotto. 
Ci sono due modi: Il primo è quello più immediato, in quanto una volta andato nella sezione CODE ti appare già la lista dei file.
Il secondo è cliccare sul "GO TO FILE". Ma meglio il primo caso. 
Clicca sul pulsante verde CODE e "Download Zip" per scaricare la traduzione.

**Comment by crazyLillo** [comment]
grazie, ci avevo gia provato e ho letto, ma adesso è molto più chiaro.
però non ho capito come faccio ad individuare quali siano i file già
tradotti e quelli ancora no.

ho un altro problema, per rispondere qui, lo devo fare tramite email,
perche non riesco ad accedere a github
inserisco nome utente e password, ma poi mi richiede un ulteriore codice
che mi inviano per email, lo inserisco,

ma mi da errore, ne richiedo un altro, lo inserisco, ma niente da fare,
errore, e così all'infinito.
suggerimenti?

**Comment by crazyLillo** [comment]
ciao tesivo, se mi leggi, non riesco ad entrare in github
io ci ho provato a venire su github, mi son registrato per dare aiuto, ma
il giorno dopo, per entrare m i richiedeva
un ulteriore codice che mi inviavano per email, ma ogni volta lo inserisco
mi da errore, così all'infinito.
( non riconosce il mio dispositivo) non so perchè, e non mi va di perderci
tempo.
ho potuto solo interagire per email, ma mi è scomodo e poi non posso
interagire, ma solo rispondere dove gia ho scritto.
Quindi mi dispiace, volevo con tutto il cuore aiutare , ma ne sono impedito.

posterò un messaggio su game translator, dove ho intenzione di lavorare li
per fare una revisione completa dei testi.
li troverai il mio aiuto se lo vuoi, io sono disponibile come ho gia detto.
ciao

**Comment by NarakuITA** [comment]
Contatta il supporto: [GitHub Support](https://support.github.com/request) loro ti sapranno aiutare a risolvere questo problema. Spiega bene il "tutto" e vediamo se ti sanno risolvere il problema. Tienimi aggiornato. In caso troviamo un altra soluzione.

**Comment by crazyLillo** [comment]
eccomi qui, grazie mille.
sono andato in support, ho visto che potevo modificare la password, e l'ho fatto, non ho compilato nulla e chiesto niente, ci ho provato ed eccomi qui.
sperando che quando mi ricollego in futuro, non mi faccia la stessa cosa.

ad ogni modo, sto quasi impazzendo (per la contentezza), sto scoprendo i mods, sdk, etc...
un sacco di roba da tradurre...
...io sono già al lavoro...sono metodico, accurato, paziente
sto ritraducendo da capo jack-tutorial, sto già al 85%, lo sto facendo direttamente sul file in inglese.
all'inizio avevo messo a confronto le due traduzioni italiane disponibili ( quella storica e quella trovata qui).
io l'ho giocato una volta sola in inglese tanto tempo fa, quello che vedo è scoraggiante e mi fa diciamo "arrabbiare"
a parte che il file in taliano e un po maltrattato e non si segue agevolmente, quello originale inglese, è molto piu ordinato
comunque ho messo a confronto le due traduzioni italiane
ragazzi, credetemi,ci sono molte frasi tradotte da strapparsi i capelli (e non esagero).

vi faccio solo 2 esempi dei tanti (non voglio sembrare l'ultimo dei pignoli

1 frase originale
Oh man, we're poppin' a cherry here! Ha ha ha! [chuckle]Ah, you're gonna love this! Alright, check it out. Blood: it's your new rack o' lamb, your new champagne

le due traduzioni a confronto
Oh ragazzo- offrirò io allora! [ridacchia] Ah, ti piacerà da morire! Bene, provalo. Il sangue è la tua nuova bevanda, il tuo nuovo champagne,
Andiamo, stai per avere la tua prima notte! Ha ha ha! [risata]Ah, ti piacera'! Bene, ascolta. Sangue: e' il tuo nuovo tormento, il tuo nuovo champagne

la mia traduzione
Oh amico, stiamo stappando uno cherry qui! Ha ha ha! [risatina beffarda] Ah, lo adorerai! Va bene, guarda. Sangue : è il tuo nuovo carré d'agnello, il tuo nuovo champagne. 

nota: lo cherry non è altro che una bottiglia di vino liquoroso

2 frase originale
[sheepishly]Uhh... shortcut. Well, nicely done though.

le due traduzioni a confronto
[belato]Uhh...scorciatoia. Beh, ben fatto.
[pecorosamente]Ahh... scorciatoia. Beh, ben fatto.

la mia traduzione
[imbarazzato]Ehm... scorciatoia. Bene, comunque ben fatto

------------------------------------------
BELATO??? - PECOROSAMENTE???
------------------------------------------

per carità non sto scrivendo questo per criticare, ma per dire che è venuto il momento di una revisione

..a volte si possono trovare altre forme ed espressioni, ma io traduco quello che leggo.

X tesivo : frasi da tradurre, io ci sono

grazie a NarakuITA, anche per te vale lo stesso, se ti serve aiuto, anche per altro, fai un fischio

**Comment by tesivo** [comment]
> ciao non si perchè man onriesco ad entrare in gitub, mi chiede un ulteriore codi che mi inviano per email, lo inserisco, ma dice errato, ne chiedo un altro, ma anche gli altri danno errore, booo ok andando al sodo... mi sta bene tutto quello che vuoi fare, io ti aiuterò nel limite del mio possibile. ma ho dei limiti per la mia obsolescenza, (quasi una 15 ina d'anni che non uso un pc dammi tempo) al tempo del commodore sproteggevo qualsiasi gioco e un po di dimestichezza mi è rimasta in certe cose sono ultaferratissimo e in altre una schiappa l'età avanza e la barba si fa sempre piu bianca per esempio, che cosè un commits? nel senso come faccio a leggerli? pensa che ci ho messo un bel po per registrarmi e capire come postare, ma c'è l'ho fatta piano piano. Allora, devi sapere che questo gioco è stata un pò la mia maledizione (ma a quanto vedo in giro, non solo mia...). la traduzione di questo gioco è sempre stata problematica per via della slang usato a Los Angeles. (io ci ho vissuto 3 anni...) sto leggendo a piu non posso in giro non sono riuscito ancora a capire, di quante patch esistano e quante mod e quali siano le migliori (magari da tradurre in futuro) ho visto le traduzioni fatte, io ne conosco almeno 3 (multi, Tir, Lunatics). francamente sono molto perplesso di moltissime stringhe tradotte alla bella e buona con aggiunte inesistenti e e ommissioni di pezzi di frase in inglese. queste per me sono solo interpretazioni di una traduzione non c'è atmosfera, non c'è pathos nelle traduzioni italiane, quella che invece c'è nel gioco in lingua originale. io se lo vuoi sapere, visto che adesso faccio parte del tuo team, ho un progetto in testa da tanto tempo. ok tradurre questo gioco a fondo una volta per tutte poi come ho detto ho un progetto tutto mio di un possibile doppiaggio soltanto dell'intero tutorial con jack. molto tempo fa ci provai a farne un pezzo in dialetto romanesco e un pezzo in napoletano stretto, era a mò di parodia, ma divertente. ma pensiamo prima alla traduzione io non so cosa è stato tradotto o no, perchè non lo sto giocando, ho fatto solo un pezzettino all'inizio per vedere se funzionava tutto. lo giocherò solo alla fine, quando sarà completo. fammi solo capire come faccio a vedere i file e i lavori di traduzione e come faccio a metterci le mani ...è arrivato il momento giusto per Bloodlines Buon Anno!

Ottimo che hai risolto con github ;)

Non preoccuparti di nulla, i commits sono, sostanzialmente, le traduzioni, effettuate, caricate nella pagina github del nostro progetto di traduzione di VTMB. Qui il link da cui puoi sfogliare le modifiche effettuate:
https://github.com/tesivo/VTMB_italian_translation_for_Unofficial_Patch/commits/master

Sì, la traduzione di VTMB è effettivamente sempre stata problematica, troppe traduzioni, delle quali nessuna completa e nessuna corretta (portando inevitabilmente alle problematiche di cui parlavi anche tu: "aggiunte inesistenti", "omissioni di pezzi di frase", "no atmosfera" e "no pathos").

Ho contattato Lunatics (edit. ci stiamo sentendo) per capire se stanno ancora traducendo il gioco(è un progetto di anni e anni fa del quale non si è visto nulla, magari hanno la migliore traduzione di tutti, ma non si sa :D) e se unire le forze. Per quanto riguarda il gruppo Tir(che mi sembra inattivo su VTBM), beh è dal loro lavoro che è partita questa traduzione. Per "multi" chi intendi? Condividimi i link che li riguardano.

L'obiettivo di questa traduzione è, (secondo me) in primis, quello di offrire una traduzione completa prima che corretta. Solo successivamente punterei a migliorarla. 
Personalmente non essendo esperto della lingua inglese, men che meno dell'americano con slang losangelino, riesco, da parte mia, solo a puntare alla completezza. Chiaramente ogni qual volta è possibile migliorare la traduzione non mi tiro indietro e carico la traduzione migliore possibile.

Quindi direi che, se tu dovessi notare traduzioni che possono essere migliorate ci saresti di grande aiuto.

Riguardo alla questione patch:
Da quanto ho capito l'ultima patch ufficiale è la "1.2". Dopodiche esistono due patch non ufficiali:

-True Patch Gold Edition, attualmente aggiornata da "Tessera", che corregge solo i bug del gioco non modificando niente del gioco. Problema: non funziona su Windows 10/11

-Unofficial Patch di "Wesp5"(che include la patch ufficiale 1.2), corregge i bug, ripristina del contenuto tagliato e aggiunge nuovi contenuti. Ovviamente se si installa la versione base della unofficial patch si ha sostanzialmente un risultato molto simile alla True Patch andando a correggere quasi solo i bug e a non stravolgere l'idea degli sviluppatori originali. Questa patch poi supporta, tra le tante migliorie e fix, il supporto al widescreen, alle mod in parallelo e, ma qui non so in che senso, il supporto al multiplayer. La versione di VTMB di Gog è già patchata con questa (edit - contiene sia la 1.2 che la Unofficial Patch 9.5 separate).

Lato mod mi trovi impreparato, però da qual che ho visto ci sono parecchie mod apprezzate, che stravolgono il titolo e lo rendono particolarmente originale.

Inizio divagazione -------------

Tieni presente che le due patch sono (erano?) in guerra tra loro. Puoi trovare sul forum ufficiale della True Patch riferimenti al fatto che la patch di Wesp5, secondo loro, non sia rispettosa della visione degli autori di VTMB, credo si riferissero alla Unofficial Patch prima della possibilità di scegliere in fase di installazione se base o plus. In ogni caso Tessera in un post sembra dire che basterebbe vedere la patch di Wesp5 come mod anzichè patch e già sarebbe corretto.

In ogni caso c'è una tale quantità di trame e sotterfugi riguardo al gioco, le sue patch, le sue patch/mod non ufficiali etc... che potrebbero farci un film. :D

Per quanto mi riguarda l'utente deve poter scegliere quale patch adottare quindi Gog sbaglia a preinstallare l'Unofficial Patch, doveva fermarsi alla patch ufficiale 1.2(edit - sembrerebbe che tiene separata l'Unofficial Patch dalla 1.2 e quindi dovrebbe essere tutto regolare). L'ideale sarebbe di poter provare entrambe le patch, ma finchè la True patch non sarà compatibile con Windows 10, sostanzialmente per presa di posizione di Tessera (da quello che ho capito), userò solo l'Unofficial Patch.

Nel dubbio consiglio la versione Steam di VTMB.

Fine divagazione -------------

Comunque sia, tornando a noi:
Ottimo che anche tu vuoi tradurlo una volta per tutte. Per il doppiaggio mi tirerei fuori, ma se serve una mano per integrare del nuovo audio in game non mancherò. (l'opzione dialetto romanesco/napoletano è particolare hihi)

Per quanto riguarda cosa manca da tradurre non è così facile spiegarsi. Potrebbe mancare poco come tanto, spesso saltano fuori traduzioni mancanti. Per non parlare di tutta la questione correttezza.

**Comment by tesivo** [comment]
> ciao tesivo, se mi leggi, non riesco ad entrare in github io ci ho provato a venire su github, mi son registrato per dare aiuto, ma il giorno dopo, per entrare m i richiedeva un ulteriore codice che mi inviavano per email, ma ogni volta lo inserisco mi da errore, così all'infinito. ( non riconosce il mio dispositivo) non so perchè, e non mi va di perderci tempo. ho potuto solo interagire per email, ma mi è scomodo e poi non posso interagire, ma solo rispondere dove gia ho scritto. Quindi mi dispiace, volevo con tutto il cuore aiutare , ma ne sono impedito. posterò un messaggio su game translator, dove ho intenzione di lavorare li per fare una revisione completa dei testi. li troverai il mio aiuto se lo vuoi, io sono disponibile come ho gia detto. ciao

In realtà se preferisci lavorare da Game Translator va bene lo stesso, chiaramente bisogna poi trovare un sistema per passarci i file e si manterrà questo sito come "contenitore" della traduzione.

**Comment by tesivo** [comment]
> eccomi qui, grazie mille. sono andato in support, ho visto che potevo modificare la password, e l'ho fatto, non ho compilato nulla e chiesto niente, ci ho provato ed eccomi qui. sperando che quando mi ricollego in futuro, non mi faccia la stessa cosa.
> 
> ad ogni modo, sto quasi impazzendo (per la contentezza), sto scoprendo i mods, sdk, etc... un sacco di roba da tradurre... ...io sono già al lavoro...sono metodico, accurato, paziente sto ritraducendo da capo jack-tutorial, sto già al 85%, lo sto facendo direttamente sul file in inglese. all'inizio avevo messo a confronto le due traduzioni italiane disponibili ( quella storica e quella trovata qui). io l'ho giocato una volta sola in inglese tanto tempo fa, quello che vedo è scoraggiante e mi fa diciamo "arrabbiare" a parte che il file in taliano e un po maltrattato e non si segue agevolmente, quello originale inglese, è molto piu ordinato comunque ho messo a confronto le due traduzioni italiane ragazzi, credetemi,ci sono molte frasi tradotte da strapparsi i capelli (e non esagero).
> 
> vi faccio solo 2 esempi dei tanti (non voglio sembrare l'ultimo dei pignoli
> 
> 1 frase originale Oh man, we're poppin' a cherry here! Ha ha ha! [chuckle]Ah, you're gonna love this! Alright, check it out. Blood: it's your new rack o' lamb, your new champagne
> 
> le due traduzioni a confronto Oh ragazzo- offrirò io allora! [ridacchia] Ah, ti piacerà da morire! Bene, provalo. Il sangue è la tua nuova bevanda, il tuo nuovo champagne, Andiamo, stai per avere la tua prima notte! Ha ha ha! [risata]Ah, ti piacera'! Bene, ascolta. Sangue: e' il tuo nuovo tormento, il tuo nuovo champagne
> 
> la mia traduzione Oh amico, stiamo stappando uno cherry qui! Ha ha ha! [risatina beffarda] Ah, lo adorerai! Va bene, guarda. Sangue : è il tuo nuovo carré d'agnello, il tuo nuovo champagne.
> 
> nota: lo cherry non è altro che una bottiglia di vino liquoroso
> 
> 2 frase originale [sheepishly]Uhh... shortcut. Well, nicely done though.
> 
> le due traduzioni a confronto [belato]Uhh...scorciatoia. Beh, ben fatto. [pecorosamente]Ahh... scorciatoia. Beh, ben fatto.
> 
> la mia traduzione [imbarazzato]Ehm... scorciatoia. Bene, comunque ben fatto

Tieni sempre presente che per ora ci occupiamo solamente del gioco patchato con la sola unofficial patch versione 10.7

Ottimo che stai effettuando una revisione/ritraduzione di jack-tutorial ;) Mi raccomando lavora dalla versione 10.7 del file inglese :)

Grazie per aver mosso una critica costruttiva alla traduzione mostrandoci dove non funziona con degli esempi e offrendoci la tua traduzione. La maggior parte delle critiche che vedevo sul web erano del tipo "Lascia perdere la traduzione, è fatta da cani. Giocalo in lingua originale che è meglio. Impara l'inglese!!!", ovviamente comprendo che si vorrebbe tutto subito e perfetto, ma bisogna anche considerare le difficoltà nel cimentarsi in un progetto così grande, così come la possibilità che qualcuno voglia giocarsi il gioco nella propria lingua a prescindere dalla sua conoscenza della lingua inglese.

Chiaramente le tue traduzioni sono decisamente più fedeli e corrette della versione inglese. Non c'è dubbio a riguardo

Chiarisco subito una cosa però: gran parte della traduzione è rimasta quella vecchia presente in rete. Non è una critica al lavoro che hanno fatto quelli prima, c'è chi non ha fatto nulla mentre loro si sono impegnati :D
L'obiettivo di questo progetto è stato, fino ad ora di, renderla compatibile con la versione 10.7 della unofficial patch, aggiungere le traduzioni mancanti e migliorare le traduzioni presenti. Avrai notato da te che la revisione della traduzione, tenendo sempre presente l'originale inglese, sarà assolutamente da svolgere. Più ci avviciniamo ad avere tradotto tutto più nasce la necessità di rivederne anche la qualità della traduzione.

Riguardo alla traduzione delle parole tra parentesi quadrate c'è da dire che avrebbe senso non tradurle, dato che sono solo delle indicazioni per i programmatori delle animazioni dei personaggi e non si vedono in gioco.

**Comment by tesivo** [comment]
Sintesi delle tre risposte:
C'è ancora molto lavoro da fare, specie nella revisione. E' giunto il suo momento :D

**Comment by NarakuITA** [comment]
Io Proporrei questo: tesivo continua con al traduzione, crazyLillo invece con la revisione. Dopo di ché una volta revisionato i file, un po' per volta li passa qui e tu (tesivo) li inserisci nel pacchetto tradotto.

**Comment by tesivo** [comment]
Si, ci sta ci sta ;)

**Comment by crazyLillo** [comment]
Cercherò di rispondere per ordine

Sto guardando i commits, ma non oso metterci le mani, potrei combinare di quei pasticci...
Per il momento mi dedico ai dialoghi del tutorial e a segnalare qualcosa qui e lì.
In futuro guardando i commits, mi darò una certa regola anche in modo da permettere di individuare il punto esatto agevolmente.

Diciamolo!, La traduzione di VTMB è stata fatta come tanti giochi in circolazione, poi magari sono stati proprio i fans a migliorarne la lettura dei sottotitoli.
Invece, Il problema della traduzione dei testi di VTMB, a parte la consistenza considerevole di testi da tradurre è la non facile interpretazione di diverse frasi e modi di dire.
Così ne uscì quello che ne uscì.

Faccio quest'esempio, perchè è una delle ultime cose, diciamo "fuori luogo" che ho visto nelle passate traduzioni.
la frase:

Leave me, Jack-of-the-box.


Lasciami in pace, Jack...(traduzione da multiplayer, quella storica la prima)

Lasciami solo, Jack-delle-scatole.( traduzione da Tir Evolution) presa qui da voi

invece, la traduzione più vera è questa :
Lasciami solo, Jack-pupazzo-nella-scatola.

per il semplice motivo che  (Jack-of-the-box) o meglio (Jack in a box) era il protagonista principale di una serie di cartoni animati orror che andava forte diversi anni fa in  L.A.
...nient'altro che un pupazzo che scattava a molla da una scatola, per ipoteticamente spaventare...
Jack-of-the-box è quindi la parodia di (Jack in a box).

Adesso direte voi, "ma pensa te, ma chi se lo immaginava..." he he he he

Rispondendo in sequenza, per Multi, intendo la traduzione storica da Multiplayer, la prima.

per quanto mi riguarda io ho installato su win 7 il gioco originale, senza mettere patch 1,2.
poi l'unofficial patch 10.7 e poi la traduzione italiana, entrambe scaricate da link di tesivo.
non ho capito cosa aggiunga la PLUS, o altre patch ma penso che piano piano ci arriveremo.
le mods, l'sdk, c'è ne di roba.
ad ogni modo io lavoro sui file originali in inglese, ma faccio molto confronti.

il doppiaggio è solo uno sfizio per passare il tempo e anche per vedere come verrebbe, per fare sul serio mi ci vorrebbe un microfono piu serio delle cuffiette col microfonino di 25 anni fa.
potrei fare diversi personaggi, anche tu tesivo, o chi volesse intraprendere questa pazza avventura del doppiaggio di VTMB.
Posso modificare le voci come e quanto voglio, in caso anche servisse doppiare personaggi femminili, ma solo fino a un certo limite. una vera voce da donna ci vorrebbe come il pane.
l'importante è avere un suono accettabilmente pulito in ingresso, poi alla magia ci penso io, vi basti pensare che oltre a manipolare il file audio, devo anche cordinare la voce con la velocità del labiale (eh si, c'è anche quello) 

le scene in primo piano poi sono parecchie anche. 
non è una cosa facile, ma si può fare. Ci vuole solo volontà e un piccolo team.
...ma ne scorrerà di acqua sotto i ponti...
Quella di farla in dialetto Romanesco è tutta una fantasia, dove si sarebbero dovute cambiare le locazioni. pensa ai 7 colli romani, i nomi tutti cambiati, personaggi tipo "banda della magliana" i modi di dire. Romanesco al 

posto dello slang.
io non sono Romano, ci ho vissuto solo un paio d'anni a Roma, in romanesco perchè è un dialetto che capiscono tutti in tutta italia e perchè è molto ironico e si presta alla perfezione. ma poi ci sarebbero mille difficoltà, è 

solo una fantasia.
fantasia chiusa.

si dovrei avere un modo per passarti del materiale, quando qui non dovrebbero funzionare le cose.
Non so, tu non vedi la mia email e comunicare per email con me, in modo che possa sapere dove mandare in caso necessità.

in verità su game translator ci sono arrivato per caso per altri motivi, dove ho letto l'appello di  NarakuITA suVTMB. il resto lo sai.

Dici le mie traduzioni sono decisamente più fedeli e corrette.  bhe! Grazie!
io ho messo a confronto le due traduzioni in mio possesso, vi ho fatto vedere solo la minima parte degli strafalcioni, e non infierisco piu di tanto perchè ormai quello è.

Si facciamo come dice NarakuITA, che in sostanza è quello che sto facendo, magari mi coordino meglio.

**Comment by tesivo** [comment]
Tranquillo, non puoi modificare ciò che è presente in questa repository di GitHub, occorre in ogni caso la mia conferma, quindi puoi fare tutte le prove che vuoi qui su GitHub (la procedura sarebbe, se ti ci vuoi cimentare, fork del progetto sul tuo profilo, modifica sul tuo profilo e quando vuoi pubblicarlo fai un pull verso questo progetto, io approvo e si crea un commit).

Si in effetti non mi aspettavo questo retroscena di Jack-of-the-box :D

Ah, la testata giornalistica Multiplayer.it intendi? 

Ok per la tua versione del gioco, considera che quando esegui il gioco con l'unofficial patch, che sia tradotta o meno, hai di fatto anche la 1.2 già applicata. La plus aggiunge degli extra, contenuto tagliato o aggiunto dai modder.

Eheh, si hanno messo anche il lip sync nel gioco

Progetto sicuramente interessante quello del doppiaggio. Per ora direi secondario, poi se qualcuno vuole cimentarsi già ora va bene, ci mancherebbe altro!

Va bene la pubblicazione delle nuove traduzioni via archivio come hai fatto nell'altra discussione (prima era un issue).

Lode a NarakuITA che ha fatto girare la voce :D

**Reply by crazyLillo** [reply]
Sono nato tranquillo, il problema è che mi sono agitato troppo dopo...

Bhe, io mi trovo meglio a lavorare direttamente sul file interessato (come ho sempre fatto, non sono proprio a digiuno riguardo traduzioni di videogames).
ci lavoro sopra e quando pronto, file compresso e lo posto qui. Per me è la cosa migliore.

Jack-of-the-box (jack in a box), se fai una ricerca, lo vedrai pure come è fatto (un pupazzo che scatta fuori a molla da una scatola).

apro parentesi
a proposito di giornalisti, nella casa a santa monica(dopo il tutorial) se accendo la TV, nei sottotitoli, il giornalista che parla In inglese "newcaster" è stato tradotto come "annunciatore radio",
ma caspita, è un televisore, è un giornalista che parla in tv, non alla radio.
chiusa parentesi.

si lo sapevo di non mettere la 1.2, perchè la unofficial ne è gia provvista.
sapevo del lip sync, ecco perchè il mio doppiaggio è sincronizzato con il labiale, ho dovuto fare in modo che il tempo della traccia sonora fosse uguale per ogni parola scandita), un lavoretto a volte non proprio facile.
per il momento è solo per cazzeggio e giusto per provare a vedere come andrebbe fatto, in caso si volesse fare qualcosina in futuro.
mi aggiungo alla Lode a NarakuITA

**Reply by tesivo** [reply]
Ottimo.

Visto il pupazzo ;)

Sì, hai ragione, tieni sempre presente che non sempre si ha ben presente il contesto del dialogo. Comunque sia grazie della segnalazione, correggo

Hai fatto bene, in sostanza il gioco e l'unofficial patch coesistono quindi se aggiorni il gioco alla 1.2 e poi installi l'unofficial patch hai semplicemente sia il gioco con la 1.2 che l'unofficial che contiene anche la 1.2, mentre se non aggiorni il gioco alla unofficial patch avrai solo l'unofficial patch che contiene la 1.2. 
Sostanzialmente se uno avvia l'unofficial patch non gli interessa aggiornare anche il gioco.

Credo, ma non ne sono sicuro, che si possa cambiare anche la durata del doppiaggio e/o delle singole parole scandite, penso che occorra cambiare gli intervalli temporali dopo le sillabe e/o la stringa del sottotitolo. Però potrebbe non essere banale fare questa modifica perchè forse occorre, in ogni caso, aggiornare tutti gli intervalli temporali presenti nel file.

**Reply by crazyLillo** [reply]
ma infatti, in precedenza avevo installato il gioco e poi la patch 1.2 e poi l'unofficial.
ma poi ho letto che con l'unofficial, non c'era bisogno di mettere la 1.2 perchè già integrata.
così ho reinstallato tutto da capo, gioco e unofficial patch 10.7.

mmmm, giocare con quei intervalli temporali potrebbe equivalere a grossi mal di testa
comunque se in seguito vorrai dedicarti al doppiaggio di qualche frase minore (intendo quelle ambientali come "crab nipple"),
con un sintetizzatore, io potrei poi prendere l'audio della frase e manipolarlo un bel po per renderlo piu credibile e accettabile.
purtroppo ho win 7, e ho visto invece che con win 10 si possono avere più pèossibilità con i sintetizzatori vocali, perchè ho visto che alcuni abbastanza sofisticati, sono solo da win 10 in poi.
